src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arr = sourceString.split(';');
+  const result = {};
+
+  arr.forEach((style) => {
+    if (style.trim()) {
+      const [key, value] = style.split(':');
+
+      result[key.trim()] = value.trim();
+    }
+  });
```

Comment:
Instead of using `forEach` to iterate and build the `result` object, consider using `reduce` which is more suited for transforming an array into a single output value, like an object. This would also align with the checklist point to avoid modifying an existing array/object directly.